packages/fxa-auth-server/lib/routes/cms.ts

```diff
@@ -39,7 +39,19 @@ export class CMSHandler {
     const entrypoint = request.query.entrypoint;
 
     try {
-      return await this.cmsManager.fetchCMSData(clientId, entrypoint);
+      const result = await this.cmsManager.fetchCMSData(clientId, entrypoint);
+
+      const { relyingParties } = result;
+      if (!relyingParties || relyingParties.length === 0) {
+        this.statsd.increment(`cms.getConfig.empty.${clientId}.${entrypoint}`);
```

Comment:
Possible to limit the `clientId` and `entrypoint` values?  Even though we validate the format of the strings, the number of possible values is still enormous.